desktop/wallet/main.js

```diff
@@ -180,33 +161,28 @@ function createWindow() {
 
     // Edit menu
     template[1].submenu.push(
-      {type: 'separator'},
+      { type: 'separator' },
       {
         label: 'Speech',
         submenu: [
-          {role: 'startspeaking'},
-          {role: 'stopspeaking'}
+          { role: 'startspeaking' },
+          { role: 'stopspeaking' }
         ]
       }
     );
 
     // Window menu
     template[3].submenu = [
-      {role: 'close'},
-      {role: 'minimize'},
-      {role: 'zoom'},
-      {type: 'separator'},
-      {role: 'front'}
+      { role: 'close' },
```

Comment:
hmm, our linters have different rules... need to add spacing rule

Reply:
I prefer with spaces for sure... how about you?

Reply:
spaces - but I use hotkey `Alt-L` (auto format in WebStorm)